dvc/repo/experiments/executor.py

```diff
@@ -41,6 +41,7 @@ def __init__(
         self.branch = branch
         self.repro_args = kwargs.pop("repro_args", [])
         self.repro_kwargs = kwargs.pop("repro_kwargs", {})
+        self.tmp_dir: Optional[TemporaryDirectory] = None
```

Comment:
Leaving up to @pmrowla  to decide what's better. I just did this to silence mypy in `logger.debug` calls made in the `Experiment`.